fedimint-server/src/consensus/engine.rs

```diff
@@ -1,4 +1,5 @@
 use std::collections::BTreeMap;
+use std::default::Default;
```

Comment:
[nit] What needed default?

Reply:
Yyyyy... I don't know, and I don't understand . :D

Reply:
It could be some glitch w.r.t merging and merge base. I don't see anything that would need it, but then ... why is clippy to failing... ?

Reply:
Did you try removing it?

Reply:
`cargo check` was still running after removing it, so I pushed an updated version without this addition.